.github/workflows/crucible-ci.yaml

```diff
@@ -19,16 +19,13 @@ jobs:
     steps:
     - uses: actions/checkout@v2
 
-    - name: ci-environment-info
-      uses: ./.github/actions/ci-environment-info
-
-    - name: Setup CI Environment
-      run: |
-        sudo ./setup-ci-environment
-
-    - name: Run fio CI
-      run: |
-        sudo ./run-ci --scenarios fio
+    - name: integration-tests
+      uses: ./.github/actions/integration-tests
+      with:
```

Comment:
how does this work (there is nothing defined below with:) ?

Reply:
I think you are looking at stale code.  Refresh.